web/src/pages/signInUp.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+import LoginFormGoogle from '../components/login-form-google';
+import LoginFormGeneral from '../components/login-form-general';
+import {Button, Container, Grid, Link, Typography} from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
+import SignUpFormGoogle from '../components/signup-form-google';
+
+const boxStyle = {
+  marginTop: '6%',
+  padding: '30px',
+  height: '100%',
+  display: 'flex',
+  alignItems: 'center',
+  flexDirection: 'Column',
+  border: '2px solid #9b9b9b',
+};
+
+class SignInUp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      login: true,
+    };
+  }
+
+  render() {
+    return (
+        this.state.login ? (
```

Comment:
Maybe you can make much more cleaner by making two functions (function that returns SignIn Container and the other one that returns SignUp container). Then, you can make something like this: `this.state.login ? {this.signin()} : {this.signup()}`